examples/mujoco/mujoco_reinforce_hl.py

```diff
@@ -29,7 +29,7 @@ def main(
     step_per_epoch: int = 30000,
     step_per_collect: int = 2048,
     repeat_per_collect: int = 1,
-    batch_size: int | None = None,
+    batch_size: int = 16,
```

Comment:
I introduced here an `int` by default because `None` would not be compatible with `batch_size` in `SampleConfig` which is always an `int`. 

If we allow `None`, there is an extra (assert) check to be done to make mypy happy.

Should we rather change the default value from 16 to 64 as it is by default in `SampleConfig` (used a couple of lines below) ?

Reply:
There is no good default value for batch size. None has a certain semantics now which was introduced with a PR not long ago. I guess sampling config should accept None and the docstring should be extended. If you want, I can do that

Reply:
> If you want, I can do that
OK.

Reply:
Done